tower-scripts/bin/cicd-control.sh

```diff
@@ -22,207 +25,256 @@ function on_exit() {
     fi
 }
 
-function is_running(){  
-  # Output to prevent ssh timeouts. Appears to timeout
-  # After about an hour of inactivity. 
-  while true; do
-    echo 
-    echo ".... $(date) ...."
-    sleep 600
-  done
-}
-
 trap on_exit EXIT
 
+function parse_and_set_vars() {
+  local var=$(echo $1 | awk -F= '{print $1}')
+
+  # Check to see if arg is valid
+  # Note: This is ugly.
+  if [[ ! " ${VALID_ARGUMENTS[*]} " == *" ${var}"* ]]; then
```

Comment:
Maybe just a simple case statement. It won't add much and would give us the ability to do some parameter validation in the future.